src/server/config.js

```diff
@@ -14,3 +20,10 @@ module.exports = {
   version: require('../../package').version,
   webpackStylesExtensions: ['css', 'less', 'sass', 'scss', 'styl']
 };
+
+// Use above config as a default one
+// Multiple other providers are available like loading config from json and more
+// Check out nconf docs for fancier examples
+nconf.defaults(config);
+
+module.exports = nconf.get();
```

Comment:
Remember to run `gulp eslint` before push :-)

/Users/este/dev/este/src/server/config.js
  29:1  warning  Newline required at end of file but not found  eol-last